services/workflows-service/src/business-report/business-report.controller.internal.ts

```diff
@@ -225,37 +225,61 @@ export class BusinessReportControllerInternal {
   @swagger.ApiOkResponse({ type: [String] })
   @swagger.ApiForbiddenResponse({ type: errors.ForbiddenException })
   @common.UsePipes(new ZodValidationPipe(ListBusinessReportsSchema, 'query'))
-  async listBusinessReports(
+  async findMany(
```

Comment:
We use the terminology of `list` in many places. `findMany` has a very strong database or repository connotation for a restful endpoint that responds with an array of a resource.

Reply:
I don't even know why it was changed... Probably while renaming something else in the service / repo...